Cargo.toml

```diff
@@ -7,8 +7,12 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+anyhow = "1"
 actix-web = "3"
 actix-rt = "2"
+config = "0.10.1"
```

Comment:
Why not using `dotenv` as it is already in the project? As we are just using `.env` files for configuration, I don't find this necessary.

`config` bring many features we are not using and for `.env` files is enough to use `dotenv` crate.

Reply:
You are right!